walnut/README.md

```diff
@@ -1,4 +1,76 @@
 # `walnut` - the O² workflow administration and linting utility
 
 The O² **w**orkflow **a**dministration and **l**i**n**ting **ut**ility is a command line program for workflow configuration
-and import validation.
\ No newline at end of file
+and import validation.
+
+## Configuration file
+
+You can check the local `walnut` configuration with
+```bash
+$ walnut about
```

Comment:
This command is not implemented, is it? Ok to keep the mention if you plan on implementing it soon after merging.

Reply:
Added in 2b52f2a